### PR TITLE
Allow `symfony/mime ^5.4` / Unlock contao 4.13 again

### PR DESCRIPTION
**Pull request by zoglo**
### Description

TL;DR:
Contao Haste in 5.2 is not installable on Contao 4.13

____

#221 introduced `symfony/mime` as a dependency, however, the minimum requirements were set to `^6.0 || ^7.0`.

Contao 4.13 needs `symfony/mime` in `^5.4`.

I've installed the change on a 4.13 with `symfony/mime` ^5.4 installed and did run the tests:
image

There is also no BC changes in the MimeTypeGuesser:
- https://github.com/symfony/mime/blob/6.4/MimeTypesInterface.php
- https://github.com/symfony/mime/blob/6.4/FileinfoMimeTypeGuesser.php
- https://github.com/symfony/mime/blob/6.4/FileBinaryMimeTypeGuesser.php

Changelog:
https://github.com/symfony/mime/blob/6.4/CHANGELOG.md

### CODE REVIEW

**Review by Toflar — APPROVED** [review]

**Comment by qzminski** [comment]
Thanks, released as 5.2.3.
